Modules/Effect/Liquid/Liquid/EmitterData.cs

```diff
@@ -83,6 +83,9 @@ public EmitterData()
 
 			OnTime = 2000;  // Milliseconds
 			OffTime = 2000; // Milliseconds	
+
+			// Default the brightness to 100%
+			Brightness = new Curve(new PointPairList(new[] { 0.0, 100.0 }, new[] { 100.0, 100.0 }));
```

Comment:
Just FYI, there is a constructor type for a Flat100 curve that does this same thing with less clutter. new Curve(CurveType.Flat100). There are defines for RampUp and RampDown also to cover the 3 typical types.